index.js

```diff
@@ -133,6 +133,7 @@ export function getRGBBytes(color, out = [0, 0, 0]) {
   out[0] = Math.round(color[0] * 255);
   out[1] = Math.round(color[1] * 255);
   out[2] = Math.round(color[2] * 255);
+  if (color[3] !== undefined) out[3] = Math.round(color[3] * 255);
```

Comment:
this function is called getRGBBytes not getRGBABytes so alpha should not be returned

Reply:
common use case is in css color with transparency `background: rgba(${getRGBBytes(color).join(',')}, 0.5)`

Reply:
So should `fromRGBBytes` and `setRGB` not return/set [r, g, b, a] as well, just [r, g, b]? Because that's the current behaviour in v1.